bin/publish_workflow_json.sh

```diff
@@ -0,0 +1,34 @@
+PR_TITLE="chore: publish workflow-ui files"
+BRANCH_NAME="publish_workflow_ui_$(date +%s)"
+
+git switch -c $BRANCH_NAME
+
+# install jq
+sudo apt-get install jq
+
+# Loop through each package, install from npm and copy the workflow-ui.json from root of the package if it exists
+cat site/plugins.json | jq ".[] | select(.workflow == true) | .package" | while read PACKAGE
+do
+  echo "Installing $PACKAGE"
+  # remove the quotes from the package name
+  PACKAGE=$(echo $PACKAGE | tr -d '"')
+  npm install $PACKAGE
+  if [ -f "node_modules/$PACKAGE/workflow-ui.json" ]; then
+    echo "Copying workflow-ui.json from $PACKAGE"
+    cp node_modules/$PACKAGE/workflow-ui.json site/$PACKAGE/workflow-ui.json
```

Comment:
Not blocking but

if you change this to `site/workflow-ui/package/$PACKAGE/workflow-ui.json` it would already accomplish the [package endpoint](https://www.notion.so/netlify/IN-PROGRESS-WorkflowUI-Spec-87cf7e47eb284c3289415ac911480aed#ea52eff83f364bd5bc31ce781456ba6e)